tools/server/server.cpp

```diff
@@ -4750,22 +4709,22 @@ int main(int argc, char ** argv) {
             return;
         }
 
-        llama_tokens tokenized_query = tokenize_input_prompts(ctx_server.vocab, query, /* add_special */ false, true)[0];
+        server_tokens tokenized_query = std::move(tokenize_input_prompts(ctx_server.vocab, ctx_server.mctx, query, /* add_special */ false, true)[0]);
```

Comment:
I think `std::move` is unnecessary here, the compiler should be good enough to optimize this